lib/primary-color.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+const BgColor = Colors.white;
+const PrimaryColor = Colors.black;
+const SecondaryColor = Color(0xff674AF9);
```

Comment:
change file name and put into folder constants,

why you import google fonts?

for constants use k for name convention "kBgColor"